fsw-react-calculator/src/Components/Buttons.js

```diff
@@ -0,0 +1,143 @@
+import { Component } from "react";
+class Buttons extends Component {
+  render() {
+    const {
+      displayString,
+      enterNum,
+      addition,
+      subtraction,
+      multiplication,
+      division,
+      posNegButton,
+      equals,
+      clear,
+    } = this.props;
+    // const displayString = Number(display.join("").toString()).toLocaleString();
+
+    return (
+      <div className="button-container">
+          
+        <h1 className="display">{displayString}</h1>
```

Comment:
I think display should be its own component and probably not a component called Buttons.

Reply:
Successfully converted display into its own component